### PR TITLE
Changes to example ISPN application

### PR DESCRIPTION
**Pull request by ian-panzica**
Add workaround for writes.
Fix issues with JF and JFS not having blocks in places and not exiting
write properly.

### CODE REVIEW

File: src/main/java/org/commonjava/util/partyline/impl/infinispan/model/FileMeta.java [hunk 1]
```diff
@@ -112,8 +120,7 @@ public String getFilePath()
         return filePath;
     }
 
-    public void writeExternal( ObjectOutput out )
-            throws IOException
+    public void writeExternal( ObjectOutput out ) throws IOException
```
Comment (jdcasey) [inline src/main/java/org/commonjava/util/partyline/impl/infinispan/model/FileMeta.java:124]: Let's include blockSize as a field we read/write in these serde methods. If we write a bunch of data and then change the block size, we don't want to render these blocks unreadable.

**Review by jdcasey — CHANGES_REQUESTED** [review]
This looks pretty good. I added one request, but it's simple.

I'd really like to see us try out simple persistence as part of the simple test example. This isn't actually clustering, but it should test as much as possible about the Infinispan implementation without setting up clustering. You can find instructions for programmatic configuration of the persistence here:

http://infinispan.org/docs/8.2.x/user_guide/user_guide.html#configuration_2

Just look for the text "The same configuration can be achieved programmatically:" in the page.

File: src/main/java/org/commonjava/util/partyline/impl/infinispan/model/FileMeta.java (hunk 1 above)
Comment (ian-panzica) [reply src/main/java/org/commonjava/util/partyline/impl/infinispan/model/FileMeta.java:124]: All set - cleaning up serialization was on my list of things todo prior.

**Comment by ian-panzica** [comment]
@jdcasey The latest commit adds cache persistence to the example. I left the other configuration in (commented out) as well. Enabling persistence helped me fix a bunch of serialization issues.

**Comment by jdcasey** [comment]
This looks pretty good @ian-panzica. Thank you again for this work!

**Review by jdcasey — APPROVED** [review]

**Pull request merged**